### ISSUE TITLE
how to install in linux?

### ISSUE THREAD

**Issue by cyberic99** [sasq64/chipmachine#66]
Hi

There are buid instructions, but how should we install chipmachine into a target folder?

Thx

**Comment by alexmyczko** [comment]
Indeed, I've managed to create a building debian package of chipmachine, however an install target is missing. And the chipmachine binary can't run without 

```
terminate called after throwing an instance of 'sol::error'
  what():  lua: error: cannot open lua/init.lua: No such file or directory
```

Patches welcome:
http://sid.ethz.ch/debian/chipmachine/NEW/

**Comment by ghost** [comment]
I used to copy directories `data` and `lua` to `/usr/share/chipmachine/` and the `chipmachine` binary from the build directory to `/usr/bin/`. But I don't know if something is missing then. So officially install instructions or a `make install` target would be nice to have.
